code/game/objects/items/reagent_containers/glass/bottle.dm

```diff
@@ -55,6 +56,23 @@
 		var/image/lid = image(icon, src, "lid_bottle")
 		overlays += lid
 
+
+
+/datum/action/item_action/glass_bottle/set_transfer_amount
+
+/datum/action/item_action/glass_bottle/set_transfer_amount/New(var/mob/living/user, var/obj/item/holder)
+	..()
+	name = "Set Transfer Amount"
+	button.name = name
+	button.overlays.Cut()
+	var/image/IMG = image('icons/obj/items/chemistry.dmi', button, "bottle-1")
```

Comment:
Not sure about the qdel, couldn't find anything in the reagent_container.dm. But otherwise I did generalise the code. I swear I tried fetching those icons ealier and that didn't work...